src/components/indicator/Indicator.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+import './Indicator.scss';
+
+type Props = {
+    children: React.ReactNode;
```

Comment:
Can this type be more specific? Since the children are rendered within a `span`, you could use `React.ReactText`. Alternatively, if you want to allow `FormattedMessages` as children, you could change the `span` to a `div` and use `React.ReactChild`.